components/admin_console/admin_definition.jsx

```diff
@@ -1975,6 +1975,14 @@ const AdminDefinition = {
                         help_text: t('admin.customization.enableSVGsDesc'),
                         help_text_default: 'Enable previews for SVG file attachments and allow them to appear in messages.',
                     },
+                    {
+                        type: Constants.SettingsTypes.TYPE_BOOL,
+                        key: 'ServiceSettings.EnableLatex',
+                        label: t('admin.customization.enableLatexTitle'),
+                        label_default: 'Enable Latex Rendering:',
+                        help_text: t('admin.customization.enableLatexDesc'),
+                        help_text_default: 'Enable rendering of Latex code.',
```

Comment:
Recommend adding more information on the disabled state of this configuration setting in the help text. 

'Enable rending of Latex code.  If false, Latex code will be highlighted only.'